src/Form/EmbedButtonForm.php

```diff
@@ -71,8 +100,8 @@ public function form(array $form, FormStateInterface $form_state) {
       $button_icon = array($file->id());
     }
 
-    $file_scheme = \Drupal::config('entity_embed.settings')->get('file_scheme');
-    $upload_directory = \Drupal::config('entity_embed.settings')->get('upload_directory');
+    $file_scheme = $this->configFactory->get('entity_embed.settings')->get('file_scheme');
```

Comment:
Lets store `configFactory->get('entity_embed.settings')` as a class member. It can be named something like `entityEmbedConfig`.